server/database/queries/order.js

```diff
@@ -0,0 +1,6 @@
+const connection = require('../config/connection');
+
+module.exports = (createdAt, totalPrice, tableNumber) => connection.query({
+  text: 'INSERT INTO user_order (create_at, total_price, table_no) VALUES ($1, $2, $3) RETURNING *;',
```

Comment:
you only need the id 
```suggestion
  text: 'INSERT INTO user_order (create_at, total_price, table_no) VALUES ($1, $2, $3) RETURNING id;',
```